composer/core/types.py

```diff
@@ -27,7 +27,10 @@
 
 Dataset = torch.utils.data.Dataset[Batch]
 
-PyTorchScheduler = torch.optim.lr_scheduler._LRScheduler
+try:
+    PyTorchScheduler = torch.optim.lr_scheduler.LRScheduler
+except:
+    PyTorchScheduler = torch.optim.lr_scheduler._LRScheduler
```

Comment:
can we leave a comment on what versions require what? eventually (likely a long time) we want to deprecate, and it's useful to know when

Reply:
done